bundles/org.openhab.binding.dominoswiss/README.md

```diff
@@ -0,0 +1,110 @@
+# Dominoswiss Binding
+
+This binding allows the controll of rollershutters, using an egate as gateway and dominoswiss radio receivers. The egate-gateway is connectet via ethernet to openhab and sends it's commands via radio to all rollershutters. See https://www.brelag.com/ for more information.
+
+
+## Supported Things
+
+EGate: Dominoswiss EGate Server. The egate is the gateway which sends the commands to the connected rollershutters. 
```

Comment:
Two things here...

Please put each sentence on its own line.
Which is the correct way? EGate vs egate, then use the correct one right through the readme.